website/docs/reference/integrations/jira-cloud-plugin-installation.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Jira Cloud Integration - Installation
+---
+
```

Comment:
Optional: you'll only need this if you use images with captions. (A lot of my sugggestions do)

```suggestion
import Figure from '@site/src/components/Figure/Figure.tsx'

```

Example of an image with a caption:
![image](https://user-images.githubusercontent.com/17786332/231954660-f7411f9f-4551-4811-8c62-4deb6d99e0d7.png)